stepper-2.0.md

```diff
@@ -142,19 +166,24 @@ When a limit pin (digital) is set to its limit state, movement in that direction
 ```
 
 **Stepper set acceleration**
-Sets the acceleration/deceleration in steps/sec^2.
+
+Sets the acceleration/deceleration in steps/sec^2. Value is specified as step/sec^2 * 1000 since Firmata doesn't support sending and receiving floats.
+
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
 2  set acceleration command                (0x08)
 3  device number                           (0-9) (Supports up to 10 motors)
-4  accel LSB                               (acceleration in steps/sec^2)
-5  accel MSB
-6  END_SYSEX                               (0xF7)
+4  accel, bits 0-6                         (acceleration in steps/sec^2 * 1000)
```

Comment:
TBD if 16 bits is sufficient for accel and also if 1000 is the best multiplier. We need to determine the min and max values to achieve the best resolution.

Anyone have any numbers?